Makefile

```diff
@@ -1,6 +1,6 @@
 DEBIAN_VERSION ?= stretch
 
-DEBIAN_VENTI_GOVERSIONS ?= 1.8 1.9
+DEBIAN_VENTI_GOVERSIONS ?= 1.8 1.9 1.10.3
```

Comment:
Can we tag the image `1.10` to use as an umbrella image (so that we can only update the image and automatically use the new version)?
I understand this will require some additional work (to specify/update the actual latest version of Go).